src/streaming/MediaPlayer.js

```diff
@@ -173,13 +175,14 @@ function MediaPlayer() {
      * @param {HTML5MediaElement=} view - Optional arg to set the video element. {@link module:MediaPlayer#attachView attachView()}
      * @param {string=} source - Optional arg to set the media source. {@link module:MediaPlayer#attachSource attachSource()}
      * @param {boolean=} AutoPlay - Optional arg to set auto play. {@link module:MediaPlayer#setAutoPlay setAutoPlay()}
+     * @param {boolean=} SkipDataLoss - Optional arg to set whether skip discontinuity in buffer when frames dropped. {@link module:MediaPlayer#setAutoPlay setAutoPlay()}
      * @see {@link module:MediaPlayer#attachView attachView()}
      * @see {@link module:MediaPlayer#attachSource attachSource()}
      * @see {@link module:MediaPlayer#setAutoPlay setAutoPlay()}
      * @memberof module:MediaPlayer
      * @instance
      */
-    function initialize(view, source, AutoPlay) {
+    function initialize(view, source, AutoPlay, SkipDataLoss) {
```

Comment:
I don't think there is a need of passing skipDataLoss property in the constructor. We can just use set method as we are doing for other properties.